source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.txt

```diff
@@ -0,0 +1,151 @@
+API:1.8.1+6967bc1b-1
```

Comment:
Nice, but don't merge this :)

Missing space after ":" here.

The rest looks good. This is sooo much better than the last version I saw of the output. Nice work :+1:

Reply:
Thanks, yes it has been a lot of back and forth though.